config/initializers/feature_toggle.rb

```diff
@@ -7,10 +7,6 @@ def self.startdate_collection_window_validation_enabled?
     Rails.env.production? || Rails.env.test? || Rails.env.staging?
   end
 
-  def self.saledate_collection_window_validation_enabled?
-    Rails.env.production? || Rails.env.test? || Rails.env.staging?
-  end
-
```

Comment:
This makes sense so that the validation could be checked, but we probably want the validation to be disabled on review app afterwards, so that 23/24 logs could be reviewed